lib/builder.js

```diff
@@ -264,7 +264,7 @@ function executeConfigFile(saveForReset, ignoreBaseURL, source) {
   };
 
   // jshint evil:true
-  new Function(source.toString()).call(global);
+  eval(source.toString());
```

Comment:
Ahh right of course this was the underlying issue. Ok, try using `eval('(function() {' + source.toString() + '})()')` here then to avoid global variable assignments, but allow local variable access to `System`?